circle.yml

```diff
@@ -2020,19 +2070,26 @@ linux-workflow: &linux-workflow
     - server-performance-tests:
         requires:
           - build
+    - system-tests-node-modules-install:
+        requires:
+          - build
     - system-tests-chrome:
         requires:
           - build
+          - system-tests-node-modules-install
```

Comment:
Since `system-tests-node-modules-install` requires `build`, lets slim this requires list to `system-tests-node-modules-install` to make the circle ci graph a bit cleaner:

Reply:
<img width="1007" alt="Screen Shot 2021-11-23 at 2 55 41 PM" src="https://user-images.githubusercontent.com/14099737/143095559-92efb948-67b9-4975-8a1a-18a6f49b93a6.png">

Reply:
oo i didn't even realize it did this, nice catch

Reply:
b6aaa99e6164c39e46dd0a62ada3f8dc6e8db681